internal/pkg/workspace/workspace.go

```diff
@@ -125,6 +125,21 @@ func (ws *Service) writeSummary(projectName string) error {
 	return ws.fsUtils.WriteFile(summaryPath, serializedWorkspaceSummary, 0644)
 }
 
+//LocalApps returns the name of all the local apps
+func (ws *Service) LocalApps() ([]string, error) {
```

Comment:
Should we call this function something like `AppNames` or `LocalAppNames`? `LocalApps` leads me to believe a list of `App` structures would be returned.

Reply:
Agreed, also a workspace is already local 😝

Reply:
done